lib/dialects/mssql/query-generator.js

```diff
@@ -685,6 +685,11 @@ var QueryGenerator = {
     if (options.limit || options.offset) {
       if (!options.order || (options.include && !orders.subQueryOrder.length)) {
         fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
+
+        if(model.name){
+          fragment += this.quoteIdentifier(model.name) + '.';
+        }
```

Comment:
There is always a `model.name`. So you do not have to test for this. And as you want to add the table name you should use `.quoteTable(model)`. That way also schemas are supported.